src/main/resources/plugin.yml

```diff
@@ -104,19 +105,13 @@ permissions:
             siegewar.town.siege.startcannonsession: true
             siegewar.town.siege.revolt.siege.start: true
             siegewar.town.siege.revolt.siege.abandon: true
+            siegewar.town.siege.revoltpeacefully: true
 
-    siegewar.command.siegewar.guide:
-        description: User is able to read guide book.
-        default: true
 
     siegewar.command.siegewar.collect:
         description: User is able to do the /siegewar collect command.
         default: true
 
-    siegewar.command.siegewar.preference:
-        description: User is able to do the /siegewar preference command.
-        default: true
-

```

Comment:
These need to be returned in to pass the tests without admins needing to specify the nodes.

Reply:
- Yes. 
- The reason for removing these two lines is because on the master branch the two keys are actually duplicated in the file.
- Screenshot from master branch: 
![image](https://user-images.githubusercontent.com/50219223/152208864-a3b5dfff-dc34-4733-b8c3-b7e12ef9e45e.png)